x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/system_indices_overwritten_callout.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+
+import { useCore } from '../../../../app_context';
+
+export const SystemIndicesOverwrittenCallOut: FunctionComponent = () => {
+  const { docLinks } = useCore();
+
+  return (
+    <EuiCallOut
+      data-test-subj="systemIndicesInfoCallOut"
+      title={i18n.translate(
+        'xpack.snapshotRestore.restoreForm.stepLogistics.systemIndicesCallOut.title',
+        {
+          defaultMessage: 'System indices will be overwritten',
+        }
+      )}
+      iconType="pin"
+      size="s"
+    >
+      <FormattedMessage
+        id="xpack.snapshotRestore.restoreForm.stepLogistics.systemIndicesCallOut.description"
+        defaultMessage="When this snapshot is restored, system indices will be overwritten with data from the snapshot. {learnMoreLink}"
```

Comment:
The original recommended text also included: `... If this is not what you want, set include_global_state to false.` This seems self-explanatory as the message only appears in the UI when global state is enabled, so I did not include it. Adding a comment here though in case anyone thinks it would be helpful to add back.